packages/test/local-server-stress-tests/src/test/localServerStressOrderSequentially.spec.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { done, isOperationType, takeAsync } from "@fluid-private/stochastic-test-utils";
+
+import { makeGenerator, reducer, saveFailures, type StressOperations } from "../baseModel.js";
+import {
+	convertToRealHandles,
+	covertLocalServerStateToDdsState,
+	DDSModelOpGenerator,
+	loadAllHandles,
+	validateConsistencyOfAllDDS,
+	type DDSModelOp,
+	type OrderSequentially,
+} from "../ddsOperations.js";
+import {
+	createLocalServerStressSuite,
+	LocalServerStressModel,
+	type LocalServerStressState,
+} from "../localServerStressHarness.js";
+
+const orderSequentiallyReducer = async (
+	state: LocalServerStressState,
+	op: OrderSequentially,
+) => {
+	const { baseModel, taggedHandles } = await loadAllHandles(state);
+	const ddsState = await covertLocalServerStateToDdsState(state);
+	const rollbackError = new Error("rollback");
+	try {
+		state.datastore.orderSequentially(() => {
+			for (const o of op.operations) {
+				baseModel.reducer(ddsState, convertToRealHandles(o, taggedHandles));
+			}
+			if (op.rollback) {
+				throw rollbackError;
```

Comment:
The intent of this combo of throw but then catch and don't throw is mysterious, at least to those of us who are newer to the concept of rollback :). A comment would help

```suggestion
				// Thowing any error during the orderSequentially callback will trigger a rollback attempt of all the ops we just played.
				// Since it's not a real error, we'll suppress it later.
				throw rollbackError;
```